framework/include/warehouses/MooseObjectWarehouseBase.h

```diff
@@ -743,6 +749,17 @@ MooseObjectWarehouseBase<T>::subdomainsCovered(std::set<SubdomainID> & subdomain
     subdomains_covered.insert(object_pair.first);
 }
 
+template <typename T>
+std::string
+MooseObjectWarehouseBase<T>::activeObjectsToFormattedString(THREAD_ID tid /*=0*/,
+                                                            const std::string & prefix) const
+{
+  std::string output;
+  for (const auto & object : _active_objects[tid])
+    output += object->name() + " ";
```

Comment:
Should use `join` here I think in order to avoid a final trailing space (unless that's what you want)

Reply:
done